### ISSUE TITLE
Re-export `bevy_egui::EguiContext` as well?

### ISSUE THREAD

**Issue by twitchyliquid64** [jakobhellermann/bevy-inspector-egui#2]
Hi!

I managed to get bevy-inspector-egui to play nicely with mouse interaction outside of the GUI, by checking `CtxRef::wants_mouse_input()` and disabling mouse interactivity for the camera, picker etc. Like this:

```rust
// system
fn data_ui(
    egui: Res<bevy_egui::EguiContext>,
    mut cameras: Query<&mut CameraRig>,
) {
    for mut c in cameras.iter_mut() {
        c.disable = egui.ctx.wants_mouse_input();
    }
}
```

The problem is that bevy-inspector-egui does not re-export `bevy_egui::EguiContext`, so I have to import the crate manually. This means that there are potentially two versions of `bevy_egui` which can get out of sync.

To support this use-case, could this crate re-export `bevy_egui::EguiContext` ? I see it already re-exports `bevy_egui::egui;`

Thanks!
Tom

**Comment by jakobhellermann** [comment]
That seems reasonable to me. I reexported the whole `bevy_egui` [here](https://github.com/jakobhellermann/bevy-inspector-egui/commit/a909496cffdebe9fd0fffb53b0563da1fff279b2).

**Comment by jakobhellermann** [comment]
The reexport is published with version 0.3.0.

**Comment by twitchyliquid64** [comment]
THANK YOU!!! ❤️
